### PR TITLE
Feature/dedication

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

**Comment by rlskoeser** [comment]
@gissoo something is still a bit off with the tree sizing/placement on desktop, but I don't think it's specific to this set of changes.

**Comment by gissoo** [comment]
@rlskoeser thank you!! A few things I noticed:
- the hover state works perfectly but I don't think you implemented [this one, the active state, for once it's clicked](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?type=design&node-id=1101%3A6020&t=0EcqVY6lqxlhQWqJ-1)
- the hover and select visual styles do not work on safari

The focus state and the interactions all work well on all browsers.

**Comment by rlskoeser** [comment]
@gissoo I've updated with the selected style, thanks for flagging.

I tried all the things I found for Safari drop shadow, but none of them worked. Not sure why; I found some examples that work in Safari, but not as complicated as what we're doing. There are a decent number of stack overflow and github issues, so we're not the only ones with this problem, but the suggestions I found and tried had no effect in Safari.

I did find that you can implement a drop shadow as an svg filter (docs: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feDropShadow), and in a quick test and i found that Safari does render this.

It's a little harder to manage the drop shadows that way, though. I wondered, would you be able to create an SVG export from Figma that has all the drop shadow filter definitions we need? (one for each leaf color, one for plaque hover, one for plaque selected).

Either way, I think we should track that separately so we can merge in the dedication and I can take account of it in the focus/accessibility updates I'm working on. Should we can make a new issue for drop-shadows on Safari?  If nothing else, we should probably have a fallback style... on iPhones are people usually on Safari?

**Comment by rlskoeser** [comment]
@gissoo do you want to check the select styles and then we can merge?

**Review by gissoo — CHANGES_REQUESTED** [review]

File: themes/timetree/assets/scss/pages/_timetree.scss [hunk 1]
```diff
@@ -210,6 +211,30 @@ body.home {
...
+        &:hover,
+        &:focus {
+          filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.5));
+          outline: none; // disable browser default focus outline box
+        }
+
+        &.select {
+          filter: drop-shadow(-6px -8px 16px #75645d)
+            drop-shadow(6px 14px 16px #75645d)
+            drop-shadow(-9px 4px 16px #75645d) drop-shadow(9px 0px 16px #75645d);
```
Comment (gissoo) [inline themes/timetree/assets/scss/pages/_timetree.scss]:
```suggestion
            drop-shadow(-9px 4px 8px #75645d) drop-shadow(9px 0px 8px #75645d);
```

File: themes/timetree/assets/scss/pages/_timetree.scss [hunk 2]
```diff
@@ -210,6 +211,30 @@ body.home {
...
+        cursor: pointer;
+
+        &:hover,
+        &:focus {
+          filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.5));
+          outline: none; // disable browser default focus outline box
+        }
+
+        &.select {
+          filter: drop-shadow(-6px -8px 16px #75645d)
```
Comment (gissoo) [inline themes/timetree/assets/scss/pages/_timetree.scss]:
```suggestion
          filter: drop-shadow(-6px -8px 8px #75645d)
```

**Comment by gissoo** [comment]
@rlskoeser yesterday when I reviewed the shadow was only around the wooden piece, right now it's around the rope as well. do you know why?

**Comment by rlskoeser** [comment]
@gissoo ok, check it out - glow now on the plaque only

**Comment by gissoo** [comment]
@rlskoeser thank you!! This looks great! 

on Safari this layer is covering the plaque permanently though, you can still select the plaque and see the dedication text, but looks wrong. 
Screenshot 2023-05-17 at 1 02 42 PM

**Comment by rlskoeser** [comment]
😬 oh no safari

**Comment by rlskoeser** [comment]
@gissoo ok, something to do with `use` vs `image` for Safari. I think it's fixed now (although no glow / drop-shadow in safari as we already knew).

**Comment by gissoo** [comment]
thank you, @rlskoeser

**Pull request merged**